Doc/whatsnew/3.10.rst

```diff
@@ -91,8 +91,8 @@ Optimizations
 =============
 
 
-Build and C API Changes
-=======================
+Build Changes
+=============
```

Comment:
I would prefer to move this section just before C API Changes: see https://docs.python.org/dev/whatsnew/3.9.html